.env.example

```
@@ -28,6 +28,11 @@ PUBLIC_DGEN_NETWORK=mainnet  # Options: mainnet, testnet, regtest
 PUBLIC_EXPLORER=https://blockstream.info
 PUBLIC_LIQUID_EXPLORER=https://blockstream.info/liquid
 
+# Breez SDK Esplora Overrides (OPTIONAL)
+# Use the backend proxy so the paid Blockstream key is applied server-side
+VITE_LIQUID_EXPLORER_URL=/api/esplora/liquid  # Production: https://your-api.railway.app/api/esplora/liquid
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)